src/utils/url.ts

```diff
@@ -31,9 +31,14 @@ export function isURLInList(url: string, list: string[]) {
 export function isURLMatched(url: string, urlTemplate: string): boolean {
     const isFirstIPV6 = url.includes('[');
```

Comment:
`isFirstIPV6` and `isSecondIPV6` should be true, if they are IPv6. So, create a `function isIPv6(url: string): boolean`, other code should stay the same.